### PR TITLE
Enable and disable a module using Kyma CLI and Busola

### PR DESCRIPTION
**Pull request by grego952**
**Description**

Changes proposed in this pull request:

- Add instruction on how to enable and disable a module using Kyma CLI and Busola

**Related issue(s)**
See also https://github.com/kyma-project/kyma/issues/17210

### CODE REVIEW

File: docs/enable_disable_module.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Your cluster comes with the Kyma custom resource (CR) already installed. It collects all metadata about the cluster, such as enabled modules, their statuses, or synchronization, using Lifecycle Manager. Lifecycle Manager uses `moduletemplate.yaml` to enable or disable modules on your cluster. 
+
+## Procedure
+
+> **NOTE:** To quickly enable a module, go to your **Cluster Details** view and select **Add Module**. Then, choose the channel of your choice and click **Add** to choose and upload your module.
+
+> **TIP:** You can also configure your Kyma CR to enable a module manually. For more details, see [Kyma CR](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/technical-reference/api/kyma-cr.md).
+
+<div tabs name="steps" group="enable-module">
```
Comment (NHingerl) [inline docs/enable_disable_module.md:15]: if this is called "enable module", why does it also contain instructions to disable it? Will readers find it? Why not two procedures (each with tabs for CLI and GUI). The "disable" procedure will be very short, but still it fulfils a separate user goal.

Comment (grego952) [reply docs/enable_disable_module.md:15]: Sure, good idea. I'll change that.

File: docs/enable_disable_module.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+title: Enable and disable a Kyma module
+---
+
+## Overview
+
+Your cluster comes with the Kyma custom resource (CR) already installed. It collects all metadata about the cluster, such as enabled modules, their statuses, or synchronization, using Lifecycle Manager. Lifecycle Manager uses `moduletemplate.yaml` to enable or disable modules on your cluster. 
+
+## Enable a module
+
+> **NOTE:** To quickly enable a module, go to your **Cluster Details** view and select **Add Module**. Then, choose the channel of your choice and click **Add** to choose and upload your module.
```
Comment (mmitoraj) [inline docs/enable_disable_module.md:11]:
```suggestion
> **TIP:** To quickly enable a module using Kyma Dashboard, go to your **Cluster Details** view and select **Add Module**. Then, select the channel of your choice and click **Add** to upload your module.
```
Maybe we should place the tip in the Kyma Dashboard tab?

File: docs/enable_disable_module.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+> **TIP:** You can also configure your Kyma CR to enable a module manually. For more details, see [Kyma CR](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/technical-reference/api/kyma-cr.md).
```
Comment (mmitoraj) [inline docs/enable_disable_module.md:13]:
```suggestion
> **TIP:** You can also configure the Kyma CR to enable a module manually. For more details, see [Kyma CR](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/technical-reference/api/kyma-cr.md).
```
I would add the TIP at the end of the ## Enable a module section. What do you think?

File: docs/enable_disable_module.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+<div tabs name="Enable a module" group="enable-disable-module">
+  <details>
+  <summary label="cli">
+  Kyma CLI
+  </summary>
+
+Check which modules are available on your cluster. Run:
```
Comment (mmitoraj) [inline docs/enable_disable_module.md:21]:
```suggestion
1. Check which modules are available on your cluster. Run: 
```

File: docs/enable_disable_module.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  <summary label="cli">
+  Kyma CLI
+  </summary>
+
+Check which modules are available on your cluster. Run: 
+   ```bash
+   kyma alpha list module
+   ```
+
+You should get a result similar to this example:
```
Comment (mmitoraj) [inline docs/enable_disable_module.md:26]:
```suggestion
   You should get a result similar to this example:
```

File: docs/enable_disable_module.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+   ```bash
+   operator.kyma-project.io/module-name    Domain Name (FQDN)         Channel     Version                      Template                     State
+        cluster-ip                   kyma-project.io/cluster-ip        fast       v0.0.24    kyma-system/moduletemplate-cluster-ip-fast   <no value>
+   ```
+
+Enable a module on your cluster in the release channel of your choice. Run:
```
Comment (mmitoraj) [inline docs/enable_disable_module.md:33]:
```suggestion
2. Enable a module on your cluster in the release channel of your choice. Run: 
```

File: docs/enable_disable_module.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+        cluster-ip                   kyma-project.io/cluster-ip        fast       v0.0.24    kyma-system/moduletemplate-cluster-ip-fast   <no value>
+   ```
+
+Enable a module on your cluster in the release channel of your choice. Run: 
+
+   ```bash
+   kyma alpha enable module {MODULE_NAME} --channel {CHANNEL_NAME} --wait
+   ```
+
+You should see the following message:
```
Comment (mmitoraj) [inline docs/enable_disable_module.md:39]:
```suggestion
   You should see the following message:
```

File: docs/enable_disable_module.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+```bash
```
Comment (mmitoraj) [inline docs/enable_disable_module.md:41]:
```suggestion
   ```bash
```

File: docs/enable_disable_module.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
+---
+title: Enable and disable a Kyma module
+---
+
+## Overview
+
+Your cluster comes with the Kyma custom resource (CR) already installed. It collects all metadata about the cluster, such as enabled modules, their statuses, or synchronization, using Lifecycle Manager. Lifecycle Manager uses `moduletemplate.yaml` to enable or disable modules on your cluster.
```
Comment (mmitoraj) [inline docs/enable_disable_module.md:7]: I wouldn't mention Lifecycle Manager, @janmedrek would you agree?

Comment (janmedrek) [reply docs/enable_disable_module.md:7]:
```suggestion
Your cluster comes with the Kyma custom resource (CR) already installed. It collects all metadata about the cluster, such as enabled modules, their statuses, or synchronization. The `moduletemplate.yaml` contains ModuleTemplate custom resource that is used to enable or disable modules on your cluster. 
```

I would go with something like this. 🙂

File: docs/enable_disable_module.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+<details>
+<summary label= Kyma Dashboard>
+Kyma Dashboard
+</summary>
+
+Follow these steps to disable a Kyma module in Kyma Dashboard:
+1. Go to the `kyma-system` Namespace.
+2. In the **Kyma** section, Choose the **Kyma** resource.
+3. Select your Kyma instance (`default-kyma`) and click **Edit**.
+4. Click on the thrash icon next to your module and update the changes.
```
Comment (mmitoraj) [inline docs/enable_disable_module.md:95]:
```suggestion
4. Click on the trash icon next to your module and update the changes.
```

File: docs/enable_disable_module.md (hunk 3 above)
Comment (NHingerl) [reply docs/enable_disable_module.md]: I second that, especially because it's ugly that we have two note/tip boxes right next to each other

**Review by pbochynski — CHANGES_REQUESTED** [review]
The procedure described here requires control plane with lifecycle-manager that synchronizes release channels. With [this decision](https://github.com/kyma-project/community/issues/775) we make clear that release channels and lifecycle-manager are suitable for managed offering (managing a large number of Kyma runtimes).
For simple use case where users want to add few Kyma modules to their kubernetes clusters the simple `kubectl apply` command for module operator would be a better choice.
I described it in [this decision record](https://github.com/kyma-project/community/issues/792)

**Comment by grego952** [comment]
> The procedure described here requires control plane with lifecycle-manager that synchronizes release channels. With [this decision](https://github.com/kyma-project/community/issues/775) we make clear that release channels and lifecycle-manager are suitable for managed offering (managing a large number of Kyma runtimes). For simple use case where users want to add few Kyma modules to their kubernetes clusters the simple `kubectl apply` command for module operator would be a better choice. I described it in [this decision record](https://github.com/kyma-project/community/issues/792)

Let's keep this doc as it is since it's more generic. We can provide a link to a landing page where the modules and their other installation steps are listed.
Each module should have a separate installation doc where the steps from the decision record are adapted.

**Pull request closed without merging**